agents/node/vcxagent-core/test/utils/faber.js

```diff
@@ -81,26 +103,46 @@ module.exports.createFaber = async function createFaber () {
     await vcxAgent.agentShutdownVcx()
   }
 
-  async function sendCredentialOffer (_revocationDetails, tailsUrl) {
+  async function createCredDef (_revocationDetails, tailsUrl) {
```

Comment:
extracted `createCredDef` out of `sendCredentialOffer`. Originally `sendCredentialOffer` was also creating schema+cred-def prior to sending credential offer.